### ISSUE TITLE
Suite hold after next task X.

### ISSUE THREAD

**Issue by ghost** [cylc/cylc-flow#32]
User request (Phil Andrews): at start-up or runtime, get cylc to hold a suite as soon as a designated task (regardless of its cycle time) finishes. This would be useful for debugging tasks as it allows easier access to task stdout and stderr, via the GUI, than using single task submit outside of the suite.

Currently you can tell a suite at start-up to stop after a designated task finishes, but not hold, or to hold after a all tasks pass a designated cycle time.

**Comment by matthewrmshin** [comment]
The simplest solution is to mimic the *stop after task* logic.

**Comment by hjoliver** [comment]
Agreed that multiple stop-after and hold-after criteria should be held, not just one.

**Comment by matthewrmshin** [comment]
This is no longer small after the final agreement.

**Comment by hjoliver** [comment]
(I thought  ultiple stop condition management would be a follow-up issue - but never mind!)

**Comment by matthewrmshin** [comment]
Would be nice if we can use an xtrigger-like syntax to do this in the graph. E.g.:

```
task1 => @hold
task2 => @stop
```
